tests/schema.py

```diff
@@ -6,6 +6,12 @@
 
 
 class CurrentVersionOFDSSchema(OFDSSchema):
+    """
+    A Schema object for libcoveofds that is used in these tests.
+    The normal Schema object always fetches the latest schema from the web.
```

Comment:
```suggestion
    The OFDSSchema object always fetches the latest schema from the web.
```

I assume that this is what's meant here?

Reply:
Yes